apps/portal/examples/dotnet-template/content/source/.env.example

```
@@ -2,11 +2,9 @@
 # Database credentials (injected by Helios Operator)
 DB_HOST=localhost
 DB_USER=postgres
-DB_PASSWORD=postgres
-DB_NAME=${{ values.name }}-db
-
-# Legacy compatibility fallback used by some templates/operators
 DB_PASS=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASS key should go before the DB_USER key

(UnorderedKey)